### ISSUE TITLE
Error on tuples with one element as keys to a dictionary

### ISSUE THREAD

**Issue by andreamatt** [microsoft/pyright#567]
Error on tuples with one element as keys to a dictionary

**Code to Reproduce**
```
from typing import Dict, Tuple

q: Dict[Tuple[str, str], int] = {
    ("a", "b"): 5
}

print(q[("a", "b")])


d: Dict[Tuple[str], int] = {
    ("a",): 5
}
print(d.get(("a",)))
print(d[("a",)])
```

**Expected behavior**
The code works just fine when executed and the assignment `("a"): 5` is parsed correctly. I'd expect the access by key to work the same as `d.get(("a",))`, i.e. no error.

![image](https://user-images.githubusercontent.com/30327952/76858360-87529080-6857-11ea-9da4-a8a5c9383117.png)

The error is on the last line
```
Argument of type 'Literal['a']' cannot be assigned to parameter 'k' of type 'Tuple[str]'
  'str' is incompatible with 'Tuple[str]'
```

I'm on Pyright extension 1.1.27

EDIT:
if the string is assigned to a typed variable
![image](https://user-images.githubusercontent.com/30327952/76862235-1793d400-685e-11ea-9751-b197dac8caf4.png)
I get the error
![image](https://user-images.githubusercontent.com/30327952/76862261-237f9600-685e-11ea-95cc-9a7d13bfff44.png)

**Comment by erictraut** [comment]
Thanks for the bug report. I'm able to repro the problem, and I'll work on a fix.

**Comment by erictraut** [comment]
I've found the cause of the bug, and the fix will be in the next published version of Pyright. Thanks again for the detailed bug report.

**Comment by andreamatt** [comment]
Wow, that was fast

**Comment by andreamatt** [comment]
When can I expect the next release?

**Comment by erictraut** [comment]
I typically publish once or twice a week, depending on my available time and the number of bug fixes and features. I'd like to get a few more fixes checked in before publishing the next release.

**Comment by erictraut** [comment]
This is now fixed in version 1.1.28, which I just published.
